whizzer/features/menu/screens/RestaurantMenu/RestaurantMenu.tsx

```diff
@@ -0,0 +1,164 @@
+import {
+  View,
+  Text,
+  StyleSheet,
+  SectionList,
+  TouchableOpacity,
+  Image,
+} from "react-native";
+import React, { useEffect, useRef, useState } from "react";
+import useSupabaseHelpers from "../../../../hooks/useSupabaseCollection";
+import { supabase } from "../../../../utils/supabase";
+import { Dish } from "../../../../utils/types";
+import CategoriesSlider from "../../components/CategoriesSlider/CategoriesSlider";
+import { RootStackScreenProps } from "../../../../types";
+import { RootScreenNames } from "../../../../navigation/constants";
+
+const ITEM_HEIGHT = 150;
+
+type RestaurantMenuScreenProps =
+  RootStackScreenProps<RootScreenNames.RestaurantMenuScreen>;
+
+const RestaurantMenuScreen: React.FC<RestaurantMenuScreenProps> = ({
+  navigation,
+  route,
+}) => {
+  const { restaurantId } = route.params;
+
+  const { data, isLoading, error } = useSupabaseHelpers(
+    supabase
+      .from<Dish>("dishes")
+      .select()
+      .filter("restaurant_id", "in", `(${restaurantId})`) // TODO: check is condition
```

Comment:
We could try to filter like this
```suggestion
      .is("restaurant_id", restaurantId)
```